src/components/AddItem/ItemDetails.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import ItemTags from "./ItemTags";
+
+function ItemDetails({ details, onDetailsChange, onTagsChange }) {
+  return (
+    <section className="item-details">
+      <div className="item-detail-container">
+        <label className="section-title">상품명</label>
+        <input
+          className="item-detail-input"
+          type="text"
+          id="itemName"
+          name="itemName"
+          placeholder="상품명을 입력해주세요"
+          value={details.itemName}
+          onChange={onDetailsChange}
+        />
+      </div>
+
+      <div className="item-detail-container">
+        <label className="section-title">상품 소개</label>
```

Comment:
label과 연관있는 요소의 아이디를 작성해주세요.

```suggestion
        <label htmlFor="itemDescription" className="section-title">상품 소개</label>
```